RELEASE/scripts/autoscend/iotms/mr2022.ash

```diff
@@ -404,9 +404,15 @@ void sweatpantsPreAdventure() {
 		}
 	}
 
-	// This is just opportunistic use of sweat. This skill should be used in auto_restore.ash.
-	if (sweat >= 95 && my_mp() < my_maxmp()) {
-		use_skill($skill[Sip Some Sweat]);
+	if (sweat >= 95) {
+		if(get_property("auto_pvpEnable").to_boolean() && spleen_left() >= 4 * (1 + item_amount($item[sweat-ade]))) {
+			// Our player participates in PVP, let's give them a low-effort spleen item to end the day with, if there's still room.
+			use_skill($skill[Make Sweat-Ade]);
```

Comment:
Would you want to consume the sweat-ade as well? Between now and end of the day, autoscend may fill spleen up with other stuff

Reply:
We discussed this on discord and it seemed most people weren't comfortable with auto-consuming PVP stuff. There may of course still be spleen items generated during the day by familiars that would later be auto-consumed by autoscend. So this only exists to generate the sweat-ade during the day, and indeed there may not be room for all of it at the end.